docs/oauth2-examples-multiresource.md

```diff
@@ -0,0 +1,174 @@
+
+    ```
+    make start-perftest-producer-with-token PRODUCER=prod_producer TOKEN=$(bin/keycloak/token prod_producer PdLHb1w8RH1oD5bpppgy8OF9G6QeRpL9)
+    ```
+
+This is an access token generated for <code>prod_producer</code>.
+```
+{
+  "exp": 1690974839,
+  "iat": 1690974539,
+  "jti": "c8edec50-5f29-4bd0-b25b-d7a46dc3474e",
+  "iss": "http://localhost:8081/realms/test",
+  "aud": "rabbit_prod",            
+  "sub": "826065e7-bb58-4b65-bbf7-8982d6cca6c8",
+  "typ": "Bearer",
+  "azp": "prod_producer",
+  "acr": "1",
+  "realm_access": {
+    "roles": [
+      "default-roles-test",
+      "offline_access",
+      "producer",
+      "uma_authorization"
+    ]
+  },
+  "resource_access": {
+    "account": {
+      "roles": [
+        "manage-account",
+        "manage-account-links",
+        "view-profile"
+      ]
+    }
+  },
+  "scope": "profile email rabbitmq.read:*/* rabbitmq.write:*/* rabbitmq.configure:*/*",
+  "clientId": "prod_producer",
+  "clientHost": "172.18.0.1",
+  "email_verified": false,
+  "preferred_username": "service-account-prod_producer",
+  "clientAddress": "172.18.0.1"
+}
+</pre>
+</li>
+<li>Similarly, launch AMQP producer <code>dev_producer</code>, registered in Keycloak too but with the permission to access <code>rabbit_dev</code> resource:
+<pre class="lang-bash">
+make start-perftest-producer-with-token PRODUCER=dev_producer TOKEN=$(bin/keycloak/token dev_producer z1PNm47wfWyulTnAaDOf1AggTy3MxX2H)
+```
+
+### Test Management UI accessed via two separate resources
+
+This is a summary of the configuration, found in [rabbitmq.conf](https://github.com/rabbitmq/rabbitmq-oauth2-tutorial/tree/support-multiple-resource-server-ids/conf/multi-keycloak/rabbitmq.conf):
+
+* There are two users declared in Keycloak: `prod_user` and `dev_user`
+* The two resources, `rabbit_prod` and `rabbit_dev`, are declared in the Rabbitmq management plugin with their own OAuth2 client (`rabbit_prod_mgt_ui` and `rabbit_dev_mgt_ui`), scopes and the label associated to each resource.
+```
+management.oauth_resource_servers.1.id = rabbit_prod
+management.oauth_resource_servers.1.client_id = rabbit_prod_mgt_ui
+management.oauth_resource_servers.1.label = RabbitMQ Production
+management.oauth_resource_servers.1.scopes = openid profile rabbitmq.tag:administrator
+
+management.oauth_resource_servers.2.id = rabbit_dev
+management.oauth_resource_servers.2.client_id = rabbit_dev_mgt_ui
+management.oauth_resource_servers.2.label = RabbitMQ Development
+management.oauth_resource_servers.2.scopes = openid profile rabbitmq.tag:management
+```
+* Because there is only one OAuth2 server, both resources share the same `provider_url`:
+<pre class="lang-ini">
+management.oauth_provider_url = http://0.0.0.0:8081/realms/test
+</pre>
+* Each OAuth2 client, `rabbit_prod_mgt_ui` and `rabbit_dev_mgt_ui`, has been declared in Keycloak so that they can only emit tokens for their respective audience, be it `rabbit_prod` and `rabbit_dev` respectively.
+
```

Comment:
some minor changes to the above: " Each OAuth2 client, `rabbit_prod_mgt_ui` and `rabbit_dev_mgt_ui` are declared in Keycloak so that they can only emit tokens for their audience `rabbit_prod` and `rabbit_dev` in this sequence: `rabbit_prod` first and then `rabbit_dev`.

I have used in sequence above if that is what you mean by "respectively"???